libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateRangePicker.tsx

```diff
@@ -150,6 +153,26 @@ class DateRangePickerComponent extends React.Component<DateRangePickerProps, IDa
             </div>
         );
 
+        const OverlayDatePicker = (
+            <Overlay
+                alignTo={`.gd-date-range-picker-${isTimeEnabled ? this.state.selectedInput : "from"}`}
```

Comment:
Just a question: Can't this be  only `.gd-date-range-picker-${this.state.selectedInput}`? I see it used as this in other places in this component.